index.js

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
+
+const core = require("@actions/core");
+const github = require("@actions/github");
+const { Octokit, App } = require("octokit");
+
+async function run() {
+    const githubToken = core.getInput("github_token");
+    const debug = core.getInput("debug") === "true" || false;
+
+    if (debug) {
+        console.log(`Event name: ${github.context.eventName}`);
+        console.log(
+            `Event payload: ${JSON.stringify(
+                github.context.payload,
+                undefined,
+                2
+            )}`
```

Comment:
You can always pass multiple values (including complex objects) to the console.
```suggestion
        console.log('Event payload:', github.context.payload);
```